src/components/PageDefinitions.js

```diff
@@ -243,16 +258,17 @@ class PageDefinitions extends AbstractPageDefinitions {
     return false
   }
 
-  async loadFromListSpec(listSpec) {
-    const { dispatch, token, definitions } = this.props
+  loadFromListSpec(listSpec) {
+    const { dispatch, definitions } = this.props
     if (listSpec.filter) this.setState({ activeFilters: listSpec.filter })
     if (listSpec.sortBy) this.setState({ activeSort: listSpec.sortBy })
     if (listSpec.sortBy || listSpec.filter) this.setState({ sequence: this.state.sequence + 1 })
 
-    const toAdd = listSpec.coordinates.map(component => EntitySpec.validateAndCreate(component)).filter(e => e)
```

Comment:
don't need to filter nulls?

Reply:
I didn't know there could be nulls here. I've added it back